static/bidder-info/smaato.yaml

```diff
@@ -16,6 +16,6 @@ capabilities:
       - native
 userSync:
   redirect:
-    url: "https://s.ad.smaato.net/c/?adExInit=p&redir={{.RedirectURL}}"
+    url: "https://s.ad.smaato.net/c/?adExInit=p&redir={{.RedirectURL}}&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}"
```

Comment:
@el-chuck Could you please validate if user sync url works? It does not seem to work for me. Here is the URL that I am trying to run - 
```
[3:54 pm] Sonali Shankar More

https://s.ad.smaato.net/c/?adExInit=p&redir=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dsmaato%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24UID&gdpr=&gdpr_consent=
```

Reply:
if the request is coming from inside eu you need to pass valid tc string with consent for smaato otherwise request will be blocked

Reply:
The request is not coming from inside eu. The request is not blocked either. It's returning 302 but the setuid endpoint is not being called. Here is the screenshot for your reference - 
![Screenshot 2023-08-18 at 10 42 25 AM](https://github.com/prebid/prebid-server/assets/8147531/4204c02f-6ff6-45aa-91e7-efcfc7e49c2c)

Reply:
i changed a configuration, please try again

Reply:
working now, thanks!